src/nrc/setup_configuration/authorization.py

```diff
@@ -0,0 +1,45 @@
+# SPDX-License-Identifier: EUPL-1.2
+# Copyright (C) 2022 Dimpact
+from django_setup_configuration.configuration import BaseConfigurationStep
+from vng_api_common.authorizations.models import AuthorizationsConfig, ComponentTypes
+from zgw_consumers.models import Service
+
+from .models import AuthorizationsConfigModel
+
+
+def get_service(slug: str) -> Service:
+    """
+    Try to find a Service and re-raise DoesNotExist with the identifier to make debugging
+    easier
+    """
+    try:
+        return Service.objects.get(slug=slug)
+    except Service.DoesNotExist as e:
+        raise Service.DoesNotExist(f"{str(e)} (identifier = {slug})")
```

Comment:
Should not you catch this exception in the AuthorizationStep.execute and raise ConfigurationRunFailed ? Or it will be caught later in the `setup_configuration` code? 
Could you please add a test for this case?

Reply:
you're right, I've changed the exception and added a test for it